File: src/sort/python/selectionsort.py
from sys import argv

def selectionSort(list):
    n = len(list)
    for i in range(1, n, 1):
        jMin = i
        for j in range(i+1, n, 1):
            if (list[j] < list[jMin]):
                jMin = j
        if (jMin != i):
            list[i], list[jMin] = list[jMin], list[i]
    return list

print(*selectionSort([ int(x) for x in argv[1:] ]))

File: src/sort/js/selectionsort.js
function selectionSort(list) {
    let n = list.length;
    for (let i = 1; i < n; i++) {
        let jMin = i
        for (let j = i+1; j < n; j++) {
            if (list[j] < list[jMin]) jMin = j
        }
        if (jMin !== i) [ list[i], list[jMin] ] = [ list[jMin], list[i] ]
    };
    return list;
}

console.log(
    selectionSort(process.argv.slice(2).map(Number)).join(" ")
)